packages/core/chrome/core-chrome-browser-internal/src/types.ts

```diff
@@ -50,5 +52,18 @@ export interface InternalChromeStart extends ChromeStart {
      * @remarks Has no effect if the chrome style is not `project`.
      */
     setSideNavComponent(component: SideNavComponent | null): void;
+
+    /**
+     * Set project breadcrumbs
+     *
+     * @param breadcrumbs
+     * @param params.absolute If true, the breadcrumbs will be set as absolute, otherwise they will be appended to the default ones. false by default.
```

Comment:
```suggestion
     * @param params.absolute If true, the breadcrumbs will replace the defaults, otherwise they will be appended to the default ones. false by default.
```